tests/EmailTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+namespace emailApp;
+include dirname(__FILE__)."/../vendor/autoload.php";
+
+class EmailTest extends \PHPUnit_Framework_TestCase {
+	
+	/**
+	 *@test
+	 */
+	public function it_can_validate_email() {
+		$object = new Email();
```

Comment:
Object name should be valid name `$object` is not a good name. Here you can use `$email`.